bodhi/client/bindings.py

```diff
@@ -858,6 +858,81 @@ def get_releases(self, **kwargs):
         """
         return self.send_request('releases/', verb='GET', params=kwargs)
 
+    @errorhandled
+    def get_critpath_packages(self, release):
+        """
+        Get a list of critical path packages for a release.
+
+        Args:
+            release (unicode): The release name to query for.
+        Returns:
+            dict: A dictionary returned by critpath_package_requests() method.
+        """
+        return self.critpath_package_request(release=release, verb='GET', auth=False, data=None)
+
+    @errorhandled
+    def add_critpath_packages(self, release, packages):
+        """
+        Add a list of critical path packages to a release.
+
+        Args:
+            release (unicode): The release name to add packages to.
+            packages (list): List of critpath package names to add.
+        Returns:
+            dict: A dictionary returned by critpath_package_requests() method.
+        """
+        package_str = ','.join(packages)
+        return self.critpath_package_request(release=release, verb='PUT', auth=True,
+                                             data={'packages': package_str})
+
+    @errorhandled
+    def remove_critpath_packages(self, release, packages):
+        """
+        Remove a list of critical path packages from a release.
+
+        Args:
+            release (unicode): The release name to remove packages from.
+            packages (list): List of critpath package names to remove.
+        Returns:
+            dict: A dictionary returned by critpath_package_requests() method.
+        """
+        package_str = ','.join(packages)
+        # DELETE request method ignores the body sometimes, so passing dictionary doesn't work
+        # However, passing it's string representation works somehow.
```

Comment:
We should identify why this is the case. Is it due to a bug in python-fedora? Is it the REST API?